Reports/Conceptual-Design.md

```diff
@@ -0,0 +1,331 @@
+
+#### Budget
+
+|Item|Cost per Item|Quantity|Total Cost for Item|
+| :- | :- | :- | :- |
+|Lithium Iron Phosphate Battery|$91.43|2|$182.86|
+|Lithium Iron Phosphate Battery Charger|$129.27|1|$129.27|
+|Total|||$312.13|
+
+#### Skills
+
+A more in-depth block diagram of the power management/distribution subsystem will be made. Also, further schematics in Altium and simulations in LTSpice will be displayed to better explain the power subsystem structure. Some hardware installation for the power circuit will be required.
+
+## Ethical, Professional, and Standards Considerations
+
+The project will have an impact on the outreach of Tennessee Tech, with the team representing the school in this competition both technically and professionally. If the team can perform well, it will build on the image of the College of Engineering for prospective students, encouraging them to attend the university and be engaged in similar endeavors in the future. In preparing for the competition, the team will follow the IEEE Code of Ethics for respectful collaboration and safe design practice. To maintain a healthy relationship among team members while also holding each other accountable, each member completes evaluations for themselves and for the others on the team, providing opportunity for self-reflection of progress and for checking in on the progress of others. 
+
+The robot movement will need to comply with Game Manual specifications. There are specific rules in place for the safety of the team and the referees/technicians at the competition. Specification 10 details the need for the robot to cease operation of all units after the given 3 minutes of play. Therefore, the motor control programming and mechanisms will need to have measures in place to quickly and safely halt the robot and any actions it is performing toward the end of the time of play. 
+
+The Game Manual provided structures for the competition to follow IEEE guidelines and general electrical standards. Resources mentioned such as datasheets and procedures correlated with clean power transmission will be followed closely. For example, the E-stop mentioned in the rules are for the safety of the competitors, judges, and audience in the case of a motor malfunction.
+
+The datasheet for the camera will be followed and the intended use matches closely with the designed use.
+The infrared laser projector is safe for use in the vicinity of humans. 
+
+Resources and materials uses for this subsystem will be built following an expected use.
+This use will be listed in their datasheets, and their datasheets will be followed properly.
+The rules point out safety measures that will be taken throughout the competition.
+A rule that sensors need to abide by is no visible strobe lights.
+Furthermore, if using non-visble lasers, lasers will need to be safe for human interaction.
+
+## Timeline
+## Statement of Contributions
```

Comment:
I need to know who will be responsible for the detailed design of each subsystem.

Reply:
Added below each subsystem's section header